platform/barefoot/sonic-platform-modules-ingrasys/debian/sonic-platform-ingrasys-s9280-64x.postinst

```diff
@@ -0,0 +1,57 @@
+# Automatically added by dh_systemd_enable
+# This will only remove masks created by d-s-h on package removal.
+deb-systemd-helper unmask s9280-64x-monitor.service >/dev/null || true
+deb-systemd-helper unmask bfn.service >/dev/null || true
+deb-systemd-helper unmask qsfp-monitor.service >/dev/null || true
+# Generate kernel modules.dep and map files for add eeprom_mb.
+depmod -a || true
+# was-enabled defaults to true, so new installations run enable.
+if deb-systemd-helper --quiet was-enabled s9280-64x-monitor.service; then
+    # Enables the unit on first installation, creates new
+    # symlinks on upgrades if the unit file has changed.
+    deb-systemd-helper enable s9280-64x-monitor.service >/dev/null || true
+else
+    # Update the statefile to add new symlinks (if any), which need to be
+    # cleaned up on purge. Also remove old symlinks.
+    deb-systemd-helper update-state s9280-64x-monitor.service >/dev/null || true
+fi
+if deb-systemd-helper --quiet was-enabled bfn.service; then
+    # Enables the unit on first installation, creates new
+    # symlinks on upgrades if the unit file has changed.
+    deb-systemd-helper enable bfn.service >/dev/null || true
+else
+    # Update the statefile to add new symlinks (if any), which need to be
+    # cleaned up on purge. Also remove old symlinks.
+    deb-systemd-helper update-state bfn.service >/dev/null || true
+fi
+if deb-systemd-helper --quiet was-enabled qsfp-monitor.service; then
+    # Enables the unit on first installation, creates new
+    # symlinks on upgrades if the unit file has changed.
+    deb-systemd-helper enable qsfp-monitor.service >/dev/null || true
+else
+    # Update the statefile to add new symlinks (if any), which need to be
+    # cleaned up on purge. Also remove old symlinks.
+    deb-systemd-helper update-state qsfp-monitor.service >/dev/null || true
+fi
+# End automatically added section
+# Automatically added by dh_installinit
+if [ -x "/etc/init.d/s9280-64x-monitor" ]; then
+    update-rc.d s9280-64x-monitor defaults >/dev/null
+    invoke-rc.d s9280-64x-monitor start || exit $?
+fi
+if [ -x "/etc/init.d/qsfp-monitor" ]; then
```

Comment:
please do not add, this is automatically generated file.

Reply:
We will remove all the automatically generated files, thanks.